<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ATMECSBLRLT-030" failures="1" tests="2" name="SampleTest" time="57.142" errors="0" timestamp="30 Nov 2020 11:23:51 GMT">
  <testcase classname="com.atmecs.SauceDemoWebsiteProject.testscript.BuyProductClass" name="buyProductTest" time="27.141">
    <failure type="java.lang.AssertionError" message="Unexpected error, please check with URL Connection timed out: connect">
      <![CDATA[java.lang.AssertionError: Unexpected error, please check with URL Connection timed out: connect
at com.atmecs.falcon.automation.ui.selenium.Browser.validateURL(Browser.java:543)
at com.atmecs.falcon.automation.ui.selenium.Browser.openURL(Browser.java:380)
at com.atmecs.SauceDemoWebsiteProject.pages.BasePage.navigate(BasePage.java:13)
at com.atmecs.SauceDemoWebsiteProject.testscript.BuyProductClass.buyProductTest(BuyProductClass.java:30)
at com.atmecs.SauceDemoWebsiteProject.runner.TestNGRunner.main(TestNGRunner.java:65)
at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- buyProductTest -->
  <testcase classname="com.atmecs.SauceDemoWebsiteProject.testscript.BuyProductClass" name="buyProductTest" time="22.558"/>
</testsuite> <!-- SampleTest -->
